minigalaxy/launcher.py

```diff
@@ -156,6 +156,8 @@ def get_final_resort_exe_cmd(game, files):
 def set_fps_display(game):
     error_message = ""
     # Enable FPS Counter for Nvidia or AMD (Mesa) users
+    if game.get_info("show_fps") == "":
```

Comment:
This seems unnecessary. In python an empty string is equal to False.

Reply:
Yeah, I was too careful, looks like it works without it.